src/config.ts

```diff
@@ -1001,6 +1001,36 @@ export default {
    */
   EME_MAX_STORED_PERSISTENT_SESSION_INFORMATION: 1000,
 
+  /**
+   * Attempts to closing a MediaKeySession can fail, most likely because the
+   * MediaKeySession was not initialized yet.
+   * When we consider that we're in one of these case, we will retry to close it.
+   *
+   * To avoid going into an infinite loop of retry, this number indicates a
+   * maximum number of attemps we're going to make (`0` meaning no retry at all,
+   * `1` only one retry and so on).
+   */
+  EME_SESSION_CLOSING_MAX_RETRY: 5,
+
+  /**
+   * When closing a MediaKeySession failed due to the reasons explained for the
+   * `EME_SESSION_CLOSING_MAX_RETRY` config property, we may (among other
+   * triggers) choose to wait a delay raising exponentially at each retry before
+   * that new attempt.
+   * This value indicates the initial value for this delay, in milliseconds.
+   */
+  EME_SESSION_CLOSING_INITIAL_DELAY: 100,
+
+  /**
+   * When closing a MediaKeySession failed due to the reasons explained for the
+   * `EME_SESSION_CLOSING_MAX_RETRY` config property, we may (among other
+   * triggers) choose to wait a delay raising exponentially at each retry before
+   * that new attempt.
+   * This value indicates the maximum possible value for this delay, in
+   * milliseconds.
+   */
+  EME_SESSION_CLOSING_MAX_DELAY: 1000,
```

Comment:
Maybe this should equal 100 * a power of two. As it is not very logical to raise from 800ms to 1000ms, if the aim of the raising delays is to wait more and more.

Reply:
I'm not sure about that because it is still important to keep this delay relatively low.
For example, it can be linked to the performance of a `loadVideo` call when the `closeSessionsOnStop` keySystems option is set.

---

I think that going:
100, 200, 400, 800, 1000, 1000, 1000 ...

Still makes sense. At first we're waiting a (still high) relatively lower delay then we raise it up until we reach a ceiling.
There's no real need for it to double every time, the point is just not to overload the API while still being able to exit that situation ASAP when possible.

And with 1 second between calls I think there shouldn't be any overloading problem (considering that there is also a max number of attempts).

We've also the exact same situation with segment requests.